src/dotnet/xlf/CommonLocalizableStrings.cs.xlf

```diff
@@ -797,6 +797,11 @@ setx PATH "%PATH%;{1}"</target>
         <target state="new">Package '{0}' is missing tool settings file DotnetToolSettings.xml.</target>
         <note />
       </trans-unit>
+      <trans-unit id="ToolPackageConflictPackageId">
+        <source>A tool with the same PackageId {0} existed.</source>
+        <target state="new">A tool with the same PackageId {0} existed.</target>
```

Comment:
Perhaps this should be `Tool '{0}' is already installed.`?

Also, do we want `dotnet install tool` to be idempotent with regards to installing a currently installed tool package? That is to say, should it return a successful exit code if the tool is already installed rather than an error?

Reply:
@KathleenDollard for this string -- >"A tool with the same PackageId {0} existed."

I just remember one talk on the internet saying passive voice is considered harmful.. but no strong opinion or opinion at all

Reply:
Changed to Tool '{0}' is already installed.